pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -284615,7 +284615,6 @@ self: {
        description = "Tunneling program over websocket protocol";
        license = lib.licenses.bsd3;
        hydraPlatforms = lib.platforms.none;
-       broken = true;
```

Comment:
Don't unmark stuff as not broken in `hackage-packages.nix` manually. This file is automatically generated, so this will be overwritten. Instead remove `wstunnel` from the list of broken packages in `configuration-hackage2nix.yaml` and target this PR against `haskell-updates`, so it will be picked up when regenerating the package set.

Reply:
Targeting haskell-updates now.